templates/ModelFile.stencil

```diff
@@ -16,7 +16,7 @@ public {{ model.objectType }} {{ model.name }} : Codable {
     {% endfor %}
     public init(
     {% for property in model.properties %}
-        {{ property.name }}: {{ property.type }}{{ property.initDefaultValue }}{% ifnot forloop.last %},{% endif%}
+        {{ property.name }}: {{ property.type }} {{ property.initDefaultValue }}{% ifnot forloop.last %},{% endif%}
```

Comment:
If we leave the trailing comma, will swiftlint or swiftformat clean it up, or does the compiler just fail? I would love to avoid having this conditional logic here is possible.

Reply:
Agreed.  But swiftformat, swiftlint does not clean them up. Compile will fail if there is trailing comma in the parameter list of initializer.